src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE student
+(
+    id         SERIAL PRIMARY KEY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name  VARCHAR(255) NOT NULL,
+    course     SMALLINT     NOT NULL
+);
+
+INSERT INTO student (first_name, last_name, course)
+values ('Natan', 'Chachko', 2),
```

Comment:
I am not sure that you need to add inserts now. 
Just imagine that in the future university in another country will buy and will start using your application. They don't want and don't need these users.